adrielgama/adrielgama.github.io
# Project Rewrite with Vite

## What the PR does

This PR rewrites the project using Vite. The parts of the project that were rewritten include the Vite configuration and the project structure.

## Why the PR is necessary

The original project was using outdated technology and rewriting it using Vite will improve the project's performance and maintainability.

## How to test the PR

To test this PR, you should navigate to the project's homepage and check if the project is functioning correctly. Additionally, you should verify that all tests are passing.

## Dependent PRs

There are no other PRs that need to be merged before this PR.
